walkthroughs/1-connecting-apps-asynchronous-messaging/walkthrough.adoc

```diff
@@ -180,6 +180,9 @@ NOTE: You are not logging into the shared Fuse Online instance, available from t
 Starting this Solution Pattern provisioned an instance of Fuse Online which is not shared with other cluster users.
 
 . Select *Customizations > API Client Connectors* from vertical navigation menu on the left.
++
```

Comment:
Normally, we add the note right after the text it pertains to instead of above it. Can you move it below and adjust the note text accordingly?

Reply:
Updated the walkthrough and put the note: at the end of 3.1.4